packages/flex-dev-utils/src/__tests__/fs.test.ts

```diff
@@ -34,6 +34,21 @@ describe('fs', () => {
     });
   });
 
+  describe('readJsonFile', () => {
+    it('should read package.json', () => {
+      const str = '{"version":1}';
```

Comment:
nitpick: you don't need this variable:

```
const readFileSync = jest.spyOn(fs.default, 'readFileSync').mockReturns('{"version":1}');`

Reply:
It's used in two of the expect() expressions below though